evmcc/evmcc_test.go

```diff
@@ -589,18 +590,18 @@ H8GZeN2ifTyJzzGo
 		Context("when a smart contract has events", func() {
 			var (
 				userCert = `-----BEGIN CERTIFICATE-----
-MIICGTCCAcCgAwIBAgIRAOdmptMzz5y0A9GOgFLxRNcwCgYIKoZIzj0EAwIwczEL
-MAkGA1UEBhMCVVMxEzARBgNVBAgTCkNhbGlmb3JuaWExFjAUBgNVBAcTDVNhbiBG
-cmFuY2lzY28xGTAXBgNVBAoTEG9yZzEuZXhhbXBsZS5jb20xHDAaBgNVBAMTE2Nh
-Lm9yZzEuZXhhbXBsZS5jb20wHhcNMTgwMjEyMDY0MDMyWhcNMjgwMjEwMDY0MDMy
-WjBbMQswCQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMN
-U2FuIEZyYW5jaXNjbzEfMB0GA1UEAwwWVXNlcjFAb3JnMS5leGFtcGxlLmNvbTBZ
-MBMGByqGSM49AgEGCCqGSM49AwEHA0IABEwsU2N6Kqrtl73S7+7/nD/LTfDFVWO4
-q3MTtbckd6MH2zTUj9idLoaQ5VNGJVTRRPs+O6bxlvl0Mitu1rcXFoyjTTBLMA4G
-A1UdDwEB/wQEAwIHgDAMBgNVHRMBAf8EAjAAMCsGA1UdIwQkMCKAIKtXuAgSGNzS
-0Yz91W08FSieahwkOU7pWJvh86pkNuxSMAoGCCqGSM49BAMCA0cAMEQCIDOGUUvv
-SgCqSQONblgBtkKuKgN36VgX+jLhZbaqMNAtAiBXiAHbgYdu3UHBVJwdTYxuFTWJ
-Vc4foA7mruwjI8sEng==
+MIICGTCCAb+gAwIBAgIQYRZkT47FyWy8I6dq5mW24zAKBggqhkjOPQQDAjBzMQsw
+CQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNU2FuIEZy
+YW5jaXNjbzEZMBcGA1UEChMQb3JnMS5leGFtcGxlLmNvbTEcMBoGA1UEAxMTY2Eu
+b3JnMS5leGFtcGxlLmNvbTAeFw0xOTA0MTIxMTI5MDBaFw0yOTA0MDkxMTI5MDBa
+MFsxCzAJBgNVBAYTAlVTMRMwEQYDVQQIEwpDYWxpZm9ybmlhMRYwFAYDVQQHEw1T
+YW4gRnJhbmNpc2NvMR8wHQYDVQQDDBZVc2VyMUBvcmcxLmV4YW1wbGUuY29tMFkw
+EwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE0OcWrcDsx3eR/8OZ1zEkWxf2RGyM0mIq
+yd/3wbnNf4XEmhteLDVAooP01CgIAviVQAFPmzxwGdErtVPWLNeiFKNNMEswDgYD
+VR0PAQH/BAQDAgeAMAwGA1UdEwEB/wQCMAAwKwYDVR0jBCQwIoAg6lRhWJYd2osx
+paEWqVMPbPEgmfr+VgEW7W3g46XFqgwwCgYIKoZIzj0EAwIDSAAwRQIhALxPONHq
+2PsQIA5cdIFznw1ROIwgPUu/v5StfSarZ7bcAiAKf3VH8X0sHFdwC/SWe+QCL4hn
+yisGs/nXC/eb2/Tt4A==
```

Comment:
does this need to change or is it debug?

Reply:
The new cert is the one that generated an error. Though now I am thinking probably doesn't need to be changed.